v2/website/pages/index.js

```diff
@@ -206,7 +210,7 @@ export default class Home extends React.Component {
           <div className={styles.sectionInner}>
             <div className={styles.row}>
               {QUOTES.map(quote => (
-                <div className={styles.column}>
+                <div key={quote.title} className={styles.column}>
```

Comment:
This would be better as `quote.name` as it's more likely that we have a collision in title (Software Engineer?) than the name. Alternatively, just use index, it's ok since QUOTES won't change.